folding-schemes/src/folding/nova/mod.rs

```diff
@@ -323,18 +323,19 @@ where
     /// CycleFold running instance
     pub cf_W_i: Witness<C2>,
     pub cf_U_i: CommittedInstance<C2>,
+    cs_is_hiding: bool,
```

Comment:
Is this parameter actually needed? It seems that it could be removed, right? (also removing it from the serialization, since it is defined at the trait level)